### DISCUSSION TITLE
jpg binary files are different

### DISCUSSION THREAD

**Discussion by airtas18** [WinMerge/winmerge#2269] [Q&A]
So I backup my phone's SD Card to an External Drive and I ran WinMerge.
It found 10 JPG files with a result that says "Binary Files are different"

Looking at the files they seem exactly the same.  What exactly is different and if they are different, why?

**Comment by sdottaka** [comment]
Could you right-click on the jpg file with the difference and perform a binary comparison as shown below to see if there is actually a difference?

Also, will the results change when compared again?

![image](https://github.com/WinMerge/winmerge/assets/98126/08fc27fd-4913-4536-8e62-82db3b38fb83)

**Comment by meteorquake** [comment]
I had a jpg the other day that was thrown up as different on a binary folder compare to a backup, but when I viewed it it was visually the same so then I did an individual binary compare between the two offending files and their binary was in fact the same.
I presumed this was just a case of a slight read error rate when working with storages, in the first case there must have been a slight read error.
Obviously non-visual differences can occur too if the file's internal extra information is updated.
